src/extension.ts

```diff
@@ -106,7 +105,7 @@ function registerCommands(uri: Uri) {
   let file = uri.toString();
   let config = workspace.getConfiguration("noir", uri);
 
-  let nargoPath = config.get<string | undefined>("nargoPath") || findNargo();
+  let nargoPath = config.get<string | undefined>("nargoPath");
```

Comment:
does that mean we no longer are looking for nargo?

Reply:
This version isn't completely fleshed out—it has many hacks. I removed `findNargo` since it uses node.js APIs and doesn't work in the browser, but you need to have separate builds for browser and desktop.